static/index.md

```diff
@@ -33,6 +33,25 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
 
 [Guide to filtering a dataset](filters/)
 
+### Rate Limiting Rules
```

Comment:
Looking at other subheading , this should be `Rate limiting rules`. L and R (lowercase)